### ISSUE TITLE
(S)CSS structure

### ISSUE THREAD

**Issue by roboshoes** [stravid/slotcars#57]
i just wanted to put out there: We need to think about the structure of the CSS (SCSS). With the asset pipe-line there are a few magic things happening when it comes to paths (even inside the `url()` of a `background-image`). Some of that magic I would consider dark and black magic or just simply annoying, since it's in your way.
I also affects in what way we have to save the images so that the CSS finds them. everything written in a SCSS file in a folder called _view_ will look for images in the same folder structure. I have a basic setup right now, which should do the trick. Depending on how much CSS there will be, we have to find a different structure.

Other than that, the css also works with the same `require` system as the JS. There are two files that are outstanding: 

**gobal.scss**
**utils.scss**

The **global.scss** is used for stayles that affect every (or most) views. Like the grass background. the **utils.scss** is for classes that are used at differetn places and don't belong to a specific view. such as overriding the style of a button. 

I hope that makes sense in some way. i will go over it again tomorrow. For now you can look into the structure: https://github.com/stravid/slotcars/tree/chore/implement-home-screen/app/assets/stylesheets

**Comment by stravid** [comment]
`utils.scss` is no valid file name, please don't use abbreviations :-)

**Comment by stravid** [comment]
You are not specifying the compressor Rails should use. Since we want CSS the file names should look like this: `filename.css.scss` instead of only `.scss`.

**Comment by stravid** [comment]
Yes there is magic happening with the Asset Pipeline. I recommend reading http://guides.rubyonrails.org/asset_pipeline.html particularly chapter 2.2.1 which should solve your path issues.

TL;DR:
You should add the ERB compressor `filename.css.scss.erb` and use the helper functions like `asset_path` instead of trying to find a valid path yourself.

**Comment by roboshoes** [comment]
Thanks for tip.

**Comment by roboshoes** [comment]
Ok - I moved the files around and use the asset pipeline with SASS helper function image-url such as `background-image: image-url("nice.jpg");`. Other than that, everything is pretty clean now.
